src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        {
+          `I am ${person.age}`
+        }
+      </p>
+    )}
+    <p className="Person__partner">
+      {isMarried(person)}
+    </p>
+  </section>
+);
+
+function isMarried(person) {
```

Comment:
```suggestion
function getMarriedLabel(person) {
```
When using the 'is' prefix in the function name 'isMarried,' we expect it to return either true or false